maps/mainmap.ctbmap

```diff
@@ -0,0 +1,139 @@
+width=60;
+height=66;
+usaArea=11,9,50,12;
+mexicoArea=11,53,50,58;
```

Comment:
usaArea and mexicoArea are defined as : x,y,width,height not x1,y1,x2,y2. See #37